syntax = "proto3";

message GetValueReq {
  int32 id = 1;
}

message GetValueRsp {
  string name = 1;
}

message SetValueReq {
  int32 id = 1;
  string name = 2;
}

message SetValueRsp {
  int32 code = 1;
}
